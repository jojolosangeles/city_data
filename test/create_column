set -x
python ../code/main.py scripts/create_column.txt > actual/create_column.out
diff expected/create_column.out actual/create_column.out
diff expected/sample20.csv actual/sample20.csv
