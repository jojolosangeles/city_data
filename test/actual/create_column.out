Test: ../pd.py
> load data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv as sample20
> columns sample20
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
> create sample20 latitude [eval(x)[0] for x in dataFrame['Location']]
sample20.latitude=
> columns sample20
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location' 'latitude']
Test completed: ../pd.py
